po/pt_BR.po

```diff
@@ -122,19 +123,16 @@ msgid "remove unreferenced items from cache"
 msgstr ""
 
 #: newsboat.cpp:128
-#, fuzzy
 msgid "Files:"
-msgstr "Arquivo: "
+msgstr "Arquivo:"
 
 #. / This is printed out by --help before the path to the config file
 #: newsboat.cpp:130
-#, fuzzy
 msgid "configuration"
-msgstr "Carregando configuração..."
+msgstr "configuração"
 
 #. / This is printed out by --help before the path to the urls file
 #: newsboat.cpp:132
-#, fuzzy
 msgid "feed URLs"
 msgstr "URLs"
```

Comment:
I think "feed" wasn't translated here.

Reply:
Right, following the current translations the msgstr should be 'URLs da fonte'